services/convert/convert.go

```diff
@@ -193,6 +195,31 @@ func ToTag(repo *repo_model.Repository, t *git.Tag) *api.Tag {
 	}
 }
 
+// ToActionTask convert a actions_model.ActionTask to an api.ActionTask
+func ToActionTask(ctx context.Context, repo *repo_model.Repository, t *actions_model.ActionTask) *api.ActionTask {
+	if err := t.LoadAttributes(ctx); err != nil {
+		panic(fmt.Sprintf("failed to execute ActionTask.LoadAttributes(): %v", err))
```

Comment:
Could we return error but not panic?

Reply:
do we prefer empty list response on error? or 500 in that case?

Reply:
I think we should return 500 and with no error detail but log the error.